summerofcode/2023.md

```diff
@@ -382,7 +382,30 @@ OCI Artifacts are a set of conventions that allows us to store assets other than
 
 ### Kubebuilder
 
+#### Leverage Phase 2 Plugins API to allow creation and usage of external plugins
```

Comment:
Could you add this to the table of contents above?

Reply:
Thanks for the  clarification! We can get this in still, just need the Table of Contents update, and then we can get these two into the system

Reply:
Added to the table of contents. Addressed in latest commit. Thanks!